pkg/scheduler/server/server.go

```diff
@@ -214,19 +214,21 @@ func (s *Server) runEtcd(ctx context.Context) error {
 
 	etcdEndpoints := clientEndpoints(s.etcdInitialPeers, s.etcdClientPorts)
 
-	c, err := etcdcron.NewEtcdMutexBuilder(clientv3.Config{Endpoints: etcdEndpoints})
+	c, err := clientv3.New(clientv3.Config{Endpoints: etcdEndpoints})
 	if err != nil {
 		return err
 	}
 
 	// pass in initial cluster endpoints, but with client ports
-	cron, err := etcdcron.New(etcdcron.WithEtcdMutexBuilder(c))
+	cron, err := etcdcron.New(
+		etcdcron.WithEtcdClient(c),
+		etcdcron.WithTriggerFunc(s.triggerJob),
+	)
```

Comment:
Question: why are we using a builder style initialization rather than the more consistent `Options`?

Reply:
Following the library's original style. We can refactor that at some point.